tests/DummyData.spec.js

```diff
@@ -18,18 +18,24 @@ Dummy.encryptionPassword = 'password';
 Dummy.keyInfo = [
     {
         userFriendlyAddress: 'NQ02 93YX MNVP HYHK 2N12 BM6B QGFG PAEN 2F55',
-        type: /** @type {1 | 2} */ (2)
+        type: /** @type {EncryptionType} */ (2)
     },
     {
         userFriendlyAddress: 'NQ71 CT4K 7R9R EHSB 7HY9 TSTP XNRQ L2RK 8U4U',
-        type: /** @type {1 | 2} */ (1)
+        type: /** @type {EncryptionType} */ (1)
     }
 ];
 
+/** @type {KeyEntry[]} */
+Dummy.keyDatabaseEntries = [
+    Object.assign({}, Dummy.keyInfo[0], {encryptedKeyPair: Dummy.encryptedKeyPairs[0]}),
+    Object.assign({}, Dummy.keyInfo[1], {encryptedKeyPair: Dummy.encryptedKeyPairs[1]})
+];
+
 /** @type {AccountInfo[]} */
 Dummy.deprecatedAccountInfo = Dummy.keyInfo.map((keyInfo, i) => ({
     userFriendlyAddress: keyInfo.userFriendlyAddress,
-    type: keyInfo.type === 1? 'low' : 'high',
+    type: keyInfo.type === EncryptionType.LOW? 'low' : 'high',
```

Comment:
Missing space before the `?`.

Reply:
Well, linting is not enabled on the test files.
Should we enable it?